Docs: Enable users to disable preview for all when publishing Kedro-Viz (#1925)
kedro-org/kedro-viz: docs/source/platform_agnostic_sharing_with_kedro_viz.md

kedro viz build
```

```{note}

> Starting from Kedro-Viz 9.1.0, previews are disabled by default for `kedro viz build`. To enable previews for all nodes of the datasets, use the `--include-preview` flag.

Reviewer: i think this might go in the next release 9.2.0

Author: Okay let me update to Kedro-Viz 9.2.0

Author: Done.

> Starting from Kedro-Viz 9.2.0, previews are disabled by default for `kedro viz build`. To enable previews for all nodes of the datasets, use the `--include-preview` flag.

Reviewer 2: ```suggestion
Starting from Kedro-Viz 9.2.0, `kedro viz build` will not include dataset previews by default. To enable previews for all the datasets, use the `--include-preview` flag.
```
